webviews/common/context.tsx

```diff
@@ -186,7 +186,7 @@ export class PRContext {
 		if (reviewers) {
 			state.reviewers = reviewers;
 		}
-		state.events = events;
+		state.events = events.length === 0 ? state.events : [...state.events, ...events];
```

Comment:
Wouldn't this work and be simpler to maintain?

```suggestion
		state.events = [...state.events, ...events];
```